src/styles/animations.css

```diff
@@ -29,3 +29,33 @@
 	animation-fill-mode: both;
 	animation-name: fadeOut;
 }
+
+@keyframes slideInDown {
+	from {
+		max-height: 0;
+	}
+	to {
+		max-height: 300px;
+	}
+}
+
+.slideInDown {
```

Comment:
Seems like they're all doing `$(node).collapsible`, but I don't think we want to go that way? 😉 

I know this solution is [not good](https://github.com/dojo/widgets/pull/26#issuecomment-278499474) and would like some more discussion on what we want to do. Are we OK with programmatic animations? Not sure there's a good way to do this in CSS alone.